views/module/comment/cdotauth.pug

```diff
@@ -37,5 +37,10 @@
             .dotDelimeter ·
             span.cact.remove Удалить
             | {{?}}
+        | {{??it.mod}}
```

Comment:
I suggest not to duplicate same actions in a different control statement, but rather refactor existing content of `.cacts` cotnainer:
1. Remove condition at line 29 and wrap "reply" action in it, as 
```
{{?it.reply}}
span.cact.reply Ответить
{{?}}
2. Rewrite conditions for "edit" and "delete" actions as `{{?it.mod || (it.reply && c.can.edit)}}`

After above changes you you will likely have a leading dot delimiter in admin/moderator view when commenting is disabled, just hide it using css (hint: we never use `.dotDelimeter` as first child).